### PR TITLE
Update deprecated-features-platform.md

### PR DESCRIPTION
**Pull request by NKarolak**
Added missing table header.

### CODE REVIEW

**Comment by christianbraeunlich** [comment]
I've seen you've already created a PR 🥇
There is another unformatted table header below the v20.0 section.

File: dev-itpro/upgrade/deprecated-features-platform.md [hunk 1]
```diff
@@ -39,6 +39,8 @@ The following configuration option will be **Removed** with [!INCLUDE[prod_short
 
 The following feature will be **Removed** with [!INCLUDE[prod_short](../developer/includes/prod_short.md)] 2023 release wave 1.
 
+|Removed or Replaced? |Why?|
+|---------|---------|
 |Removed | Support for the file format .flf for Business Central license files for on-premises deployments will be removed in a later release. Please transition to using the newer .bclicense format instead. |
```
Comment (christianbraeunlich) [inline dev-itpro/upgrade/deprecated-features-platform.md:44]:
```suggestion
|Removed | Support for the file format .flf for Business Central license files for on-premises deployments will be removed in the 2023 release wave 1 (version 22.0). Please transition to using the newer .bclicense format instead. |
```

**Comment by NKarolak** [comment]
> But one thing I'd like to figure out: "Is it being **removed** or rather **replaced** (by .bclicense) ?"

I don't change any content here. It's on Microsoft to do so. Hence I only added the missing header.

**Comment by edupont04** [comment]
Thank you for your contribution! The .flf format is being deprecated as communicated here and in the release plan.

**Pull request merged**